### PR TITLE
Fix bug in GeoLib::computeRotationMatrixToXY

### PR DESCRIPTION
**Pull request by TomFischer**
As titled. Also added a simple test.

### CODE REVIEW

File: Tests/GeoLib/TestComputeRotationMatrix.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#include "gtest/gtest.h"
+#include "AnalyticalGeometry.h"
+
+TEST(GeoLib, ComputeRotationMatrixToXY)
+{
+	MathLib::Vector3 const n(0.0, -1.0, 0.0);
+	MathLib::DenseMatrix<double> rot_mat(3,3,0.0);
+
+	GeoLib::computeRotationMatrixToXY(n, rot_mat);
+	EXPECT_EQ(1.0, rot_mat(0,0));
```
Comment (norihiro-w) [inline Tests/GeoLib/TestComputeRotationMatrix.cpp]: Can you take into account  a plane normal vector even for a XZ plane? The expected rotation matrix here is actually for the case of a normal vector (0, 1, 0). If a normal vector is (0, -1, 0), I would expect a matrix like

```
1 0 0
0 0 1
0 -1 0
```

Comment (TomFischer) [inline Tests/GeoLib/TestComputeRotationMatrix.cpp]: Added test for this case and handle it in `computeRotationMatrixToXY`. Changed the commits, respectively.

Comment (norihiro-w) [inline Tests/GeoLib/TestComputeRotationMatrix.cpp]: thanks, it worked!

**Comment by norihiro-w** [comment]
:+1:

**Pull request merged**